### PR TITLE
chore: Improve docker

### PR DESCRIPTION
**Pull request by bushidocodes**
Various Improvements:
* Change docker user from root to node to make more secure
* Add nodemon (auto refresh of node when JS files are changed)
* Update `commands.md` to show new command to develop with

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,9 @@
     "test": "echo \"Error: no test specified\" && exit 1",
     "cleanData": "node cleanData.js",
     "server": "node node.js",
-    "client": "node client.js"
+    "client": "nodemon client.js",
```
Comment (albero94) [inline package.json]: Good functionality to have the ability to restart more efficiently.

**Review by albero94 — COMMENTED** [review]
I don't have the ability yet to test the docker piece, so maybe someone else should review this part.

**Comment by heidiklum** [comment]
Ignoring issue whereby the numerical ID doesn't match the arbitrarily assigned alphanumeric friendly name.

**Review by heidiklum — APPROVED** [review]
Ignoring issue whereby the numerical ID (as in --id) is different than the arbitrarily assigned alphanumeric friendly name (as in node0).

**Pull request merged**